m2c/asm_file.py

```diff
@@ -337,7 +337,7 @@ def re_comment_replacer(match: Match[str]) -> str:
     re_comment_or_string = re.compile(r'[#;].*|/\*.*?\*/|"(?:\\.|[^\\"])*"')
     re_whitespace_or_string = re.compile(r'\s+|"(?:\\.|[^\\"])*"')
     re_local_glabel = re.compile("L(_.*_)?[0-9A-F]{8}")
-    re_local_label = re.compile("loc_|locret_|def_|lbl_|LAB_|jump_")
+    re_local_label = re.compile("loc_|locret_|def_|lbl_|LAB_|jump_|.L")
```

Comment:
Need to escape the `.` or it will serve as a wildcard.
```suggestion
    re_local_label = re.compile(r"loc_|locret_|def_|lbl_|LAB_|jump_|\.L")
```